app/main.py

```diff
@@ -1,3 +1,14 @@
 def cache(func):
-    # Write your code here
-    pass
+    dict_with_args = {}
+
+    def wrapper(*args):
+        nonlocal dict_with_args
+        if args in dict_with_args:
+            print("Getting from cache")
+            return dict_with_args[args]
+
+        dict_with_args.update({args: func(*args)})
+        print("Calculating new result")
+        return func
```

Comment:
Here you should return a calculated result, not a `func` itself.
Example how it works:
```
@cache
def add(a, b):
    return a + b


print(add(2, 3))
```
Output:
```
Calculating new result
<function add at 0x1009a3e50>
```
How it should work:
```
Calculating new result
5
```